wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md

```diff
@@ -0,0 +1,142 @@
+           * American timezone confirmed (Arielle)
+           * Should ask Shern/Saranjeet for planning committee for earlier time zones
+   * Book Dash Nov2023 report PR review: [https://github.com/the-turing-way/the-turing-way/pull/3503](https://github.com/the-turing-way/the-turing-way/pull/3503)
+       * Should Kirstie/Malvika merge
+       * Esther to check 
+   * Transparency of their work - should we open a repo?
+       * Infrastructure, accessibility, newsletter does have a repo
+       * Have a bookdash repository for WG and planning committee
+       * Emma/Ale to set it up
+   * How they can allow others to see their work
+   * Chapter in the Community Handbook
+       * Need to include new information on the WG (next session / Feb)
+   * Use of issue/discussing on main repoREADME, landing page (both in book and .github): 
+
+
+### Timeline / Change dates to May 2024
+
+   * Calls for applications to join the Book Dash Planning Committee and host local hubs open: 15 Feb 2024 (3 months before)
+   * Deadline to complete the Expression of Interest to Join *The Turing Way* Book Dash Committee: 1 March 2024 (plus community outreach)
+   * Deadline for local hub - an expression of interest: 1 March 2024
+   * ALE WILL UPDATE THEM / suggestion
+   * Call for Applications for Participants: Tuesday 5 March 2024```
+   * Deadline for submission: Monday 15 April (anywhere on Earth) ```
+   * Decisions on the applications: Latest by 01 May 2024
+   * Pre Book Dash Onboarding calls (2 x 1 hour): week beginning 22 May, exact date TBC
+       * 9:00 am session: led Batool, Saranjeet, Emma, Anne, Alexandra
+       * 17:00 session: Arielle, Anne, Alexandra
+   * Pre Book Dash GitHub Skill-up (1 hour): May 2024, led by Batool Almarzouq? and Johanna Bayer?
+   * Book Dash Contribution Sessions during the week: of 3-6 June 2024
+   * The Turing Way community share-out: 7 June 2024. 
+*Curated from the DAY-5 share out notes*
+
+
+
+## Notes
+
+* Idea of the WG is to be strategic, whereas the committee is operational
+* We need more gender diversity in the planning committee so that the WG can then also become more diverse. 
+* Candidates for community outreach for planning committee membership:
+
+    - Richard Acton 
+
+    - Alejandro Coca-Castro 
+
+    - David Llewllyn-Jones
+
+    - Jim Madge 
+
+    - Kalle Westerling
+
+    - Andrea Sanchez-Tapia
+
+    - Alden Conner
+
+    - Batool Almarzouq
+
+    - Johanna Bayer
+
+    - Liz Hare
+
+    - Saranjeet Kaur
+
+    - Winny Nekesa
+
+###  Decisions / Action Items
+
+   * Invite Susana and Carlos to join WG in first instance - Esther (done)
+   * There is a need for a more balanced ratio of genders in the Book Dash planning committee
+   * Community outreach to previous planning committee members and specific Book Dash attendees for the next planning committee applications - to be divided up 
+   * First Book Dash of 2024 will be 13-17 May 2024 
```

Comment:
After some Slacks discussions, the agreed dates for the first Book Dash are 3-7 June 2024